kafka/conn.py

```diff
@@ -877,7 +878,9 @@ def _check_api_version_response(self, response):
         # in descending order. As soon as we find one that works, return it
         test_cases = [
             # format (<broker verion>, <needed struct>)
-            ((0, 10, 1), MetadataRequest[2])
+            ((0, 10, 1), MetadataRequest[2]),
+            ((0, 10, 2), OffsetFetchRequest[2]),
+            ((0, 11, 0), FetchRequest[5]),
```

Comment:
Well it's not really something that was introduced in this PR, I just extended it with new checks. I did propose using ApiVersion response for min-max per-broker checks on protocol versions in  https://github.com/dpkp/kafka-python/issues/865. Here we just determine `api_version` like we used to in versions before, only for v0.10 and above we don't do additional requests, only check that max version for a protocol is supported.
I'll change the comment to be clear on ordering.